src/webui/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
And I wonder why your files do not have trailing new line even this is set to `true`...

Reply:
but `.tsx .scss` these files all have new line in last line.